app/views/dialogue.py

```diff
@@ -466,3 +459,118 @@ def _setup_messagebox(title: str | None) -> QMessageBox:
         dialogue.setWindowTitle(DEFAULT_TITLE)
 
     return dialogue
+
+
+def show_settings_error() -> None:
+    """
+    Displays a fatal error box indicating the app was
+    unable to start due to corrupt settings. Called if unable
+    to parse the settings file.
+
+    :param settings: settings model to allow resetting settings
+    """
+    logger.info("Showing settings failure box")
+    diag = SettingsFailureDialog()
+    diag.exec_()
+
+
+class SettingsFailureDialog(QDialog):
+    """Custom dialog to display fatal errors regarding settings parsing.
+
+    Has buttons to open the settings file, open the settings folder, or reset settings.
+    Exiting the dialog will terminate the application.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        # Set up the message box
+        self.setWindowTitle("Unable to parse settings file!")
+        self.setModal(True)
+        self.setObjectName("dialogue")
+
+        # Dynamic sizing
+        self.setSizePolicy(
+            QSizePolicy(QSizePolicy.Policy.Expanding, QSizePolicy.Policy.Expanding)
+        )
+
+        # Add data
+        self.text = "Your RimSort settings file is corrupt.\nPlease choose one of the following options to proceed."
+
+        # Buttons
+        self.open_settings_file_btn = QPushButton("Open Settings File")
```

Comment:
Just a nickpick and up to your discretion, but since the dialogue is a little crowded, we may want to differentiate better between "Open Settings File" and "Open Settings Folder", especially because the stings are the same up to "Open Settings F". Maybe just "Open Settings"?

Reply:
Revised, thanks for the suggestion